integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/pages/SupervisorMessagePage.groovy

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.cattool.pages
+
+import geb.Page
+
+class SupervisorMessagePage extends Page {
+
+  static url = '/form/supervisor/supervisorMessage'
+
+  static at = {
+    headingText == 'Message from supervisor'
+  }
+
+  static content = {
+    headingText { $('h1.govuk-heading-l').text() }
```

Comment:
not sure about all this selecting elements based on govuk styles, makes the tests very brittle if govuk change...